solutions/devicesimulation-nohub/single-vm/setup.sh

```diff
@@ -50,6 +50,7 @@ while [ "$#" -gt 0 ]; do
         --aad-appid)               PCS_WEBUI_AUTH_AAD_APPID="$2" ;;
         --aad-tenant)              PCS_WEBUI_AUTH_AAD_TENANT="$2" ;;
         --aad-instance)            PCS_WEBUI_AUTH_AAD_INSTANCE="$2" ;;
+        --deployment-id)           PCS_SOLUTION_ID="$2" ;;
```

Comment:
why different 2 names for the same value?

Reply:
should be solution id, missed updating this.